packages/beacon-node/src/execution/engine/types.ts

```diff
@@ -82,6 +85,22 @@ export type EngineApiRpcParamTypes = {
   engine_getBlobsV1: [DATA[]];
 };
 
+// Can extend keys to other engine API whenever it sees fit
+export type EngineApiParamTypes = {
```

Comment:
so while this type now makes the function call type safe it still offers the possibility that the type is not updated correctly, I was playing around with this locally and though to myself adding another type is not acceptable